app/components/support_interface/personal_information_component.rb

```diff
@@ -142,7 +142,7 @@ def editable?
     end
 
     def visa_or_immigration_status_text
-      includes_eu_eea_swiss?(application_form.nationalities) ? 'Immigration status' : 'Visa or immigration status'
+      'Visa or immigration status'
```

Comment:
Can we move to a constant or locale?

Reply:
Sure, updated here: https://github.com/DFE-Digital/apply-for-teacher-training/pull/9273/commits/d4224dfb04da386c980543868f42b416cfc1f0d7